h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostTest.java

```diff
@@ -1270,4 +1284,31 @@ private static XGBoostMojoModel getMojo(XGBoostModel model) throws IOException {
     return (XGBoostMojoModel) MojoModel.load(mojoReaderBackend);
   }
 
+  private static <T extends ModelMetricsSupervised> void checkMetrics(final T expectedMM, final T actualMM) {
```

Comment:
:+1: For extracting the logic into this method. Reusable across different tests.